docs/dev/table/sql.md

```diff
@@ -1842,6 +1842,16 @@ RPAD(text string, len integer, pad string)
     <tr>
       <td>
         {% highlight text %}
+ASCII(text string)
+{% endhighlight %}
+      </td>
+      <td>
+        <p>Returns the numeric value of the leftmost character of the string str. Returns 0 if str is the empty string. Returns NULL if str is NULL. E.g. <code>ASCII('THIS')</code> returns <code>84</code>.</p>
```

Comment:
Please do not copy documentation from other projects. Especially if other projects are not Apache licensed. Even worse than copying is copying but not adjusting variable names. You use `text string` in the paramter but `str` in the description.